ts/features/idpay/payment/screens/IDPayPaymentAuthorizationScreen.tsx

```diff
@@ -59,39 +69,207 @@ const IDPayPaymentAuthorizationScreen = () => {
 };
 
-export type { IDPayPaymentAuthorizationScreenRouteParams };
+const ExistingTransactionDataContent = ({
+  data
+}: {
+  data: NonNullable<AuthPaymentResponseDTO>;
+}) => {
+  const getValueOrDash = (value: string | undefined) =>
+    pipe(
+      value,
+      O.fromNullable,
+      O.getOrElse(() => "-")
+    );
+  const amountCents = formatNumberCurrency(data.amountCents / 100);
+  const reward = pipe(
+    data.reward,
+    O.fromNullable,
+    O.fold(
+      () => "-",
+      data => formatNumberCurrency(data / 100)
+    )
+  );
+  const businessName = getValueOrDash(data.businessName);
+  const initiativeName = getValueOrDash(data.initiativeName);
+  const date = pipe(
+    data.trxDate,
+    O.fromNullable,
+    O.fold(
+      () => "-",
+      date => format(new Date(date), "D MMMM YYYY, HH:mm")
+    )
+  );
+  return (
+    <>
+      <Divider />
+      <VSpacer size={16} />
+      <View style={[IOStyles.rowSpaceBetween, IOStyles.alignCenter]}>
+        <NewH6>{I18n.t("idpay.payment.authorization.toAuth")}</NewH6>
+        <H1>{reward}</H1>
+      </View>
+
+      <VSpacer size={16} />
+      <Divider />
+      <ListItemInfo
+        label={I18n.t("idpay.payment.authorization.amount")}
+        value={amountCents}
+        accessibilityLabel=" "
+      />
+      <Divider />
+      <ListItemInfo
+        label={I18n.t("idpay.payment.authorization.businessName")}
+        value={businessName}
+        accessibilityLabel=" "
+      />
+      <Divider />
+      <ListItemInfo
+        label={I18n.t("idpay.payment.authorization.dateTime")}
+        value={date}
+        accessibilityLabel=" "
+      />
+
+      <VSpacer size={24} />
+      {/* TODO:: will be removed in favor of LIST_GROUP_HEADING in future updates */}
+      <View style={[IOStyles.row, IOStyles.alignCenter]}>
+        <Icon name="initiatives" size={24} color="bluegrey" />
+        <HSpacer size={16} />
+        <H3
+          color="bluegrey"
+          style={{
+            // this should not happen, but the current typography adds 4
+            // to paddingBottom because of line height
+            // so we add 4 to paddingTop to compensate, else the text would not be centered
+            // (this was temporarily approved by @dmnplb)
+            paddingTop: 4
+          }}
+        >
+          {I18n.t("idpay.payment.authorization.infoDivider")}
+        </H3>
+      </View>
+      <VSpacer size={16} />
+
+      <ListItemInfo
+        label={I18n.t("idpay.payment.authorization.initiativeName")}
+        value={initiativeName}
+        accessibilityLabel=" "
+      />
+      <Divider />
+      <ListItemInfo
+        label={I18n.t("idpay.payment.authorization.availableAmount")}
+        value={"-"}
+        accessibilityLabel=" "
+      />
+    </>
+  );
+};
+
+const SmallPlaceHolder = () => (
+  <Placeholder.Box animate="fade" height={16} width={100} radius={8} />
+);
+const BigPlaceHolder = () => (
+  <Placeholder.Box animate="fade" height={29} width={100} />
+);
```

Comment:
We have the `Skeleton` component :)

https://github.com/pagopa/io-app/blob/master/ts/features/idpay/common/components/Skeleton.tsx